view_w.php

```diff
@@ -39,7 +39,7 @@
 $nextStr = translate ( 'Next' );
 $prevStr = translate ( 'Previous' );
 
-$can_add = ( empty ( $ADD_LINK_IN_VIEWS ) || $ADD_LINK_IN_VIEWS != 'N' );
+$can_add = ( $ADD_LINK_IN_VIEWS === 'Y' );
```

Comment:
Speaking of the
// $Id: view_w.php,v 1.81 2009/11/22 22:26:18 bbannon Exp $
line. Since GitHub doesn't track version numbers, and doesn't expand $Id$ into anything useful, is there any point in keeping these lines in the code?